spec/example_app/db/schema.rb

```diff
@@ -59,9 +59,10 @@
     t.string   "address_city"
     t.string   "address_state"
     t.string   "address_zip"
-    t.datetime "created_at",       null: false
-    t.datetime "updated_at",       null: false
+    t.datetime "created_at",                   null: false
+    t.datetime "updated_at",                   null: false
     t.datetime "shipped_at"
+    t.integer  "status",           default: 0, null: false
```

Comment:
yeah, you can just ignore that. I opened up https://github.com/thoughtbot/administrate/pull/336 to fix this in the future.